packages/server/src/service/gm.ts

```diff
@@ -56,8 +50,8 @@ const rawZipSearch = async (zip: string): Promise<google.maps.GeocoderResult | n
   // Because maps API uses ccTLD (https://developers.google.com/maps/documentation/geocoding/overview#RegionCodes)
   // since .io is assigned to the British Indian Ocean Territory we ensure
   // the usage of component filtering to avoid issues related to ccTLD.
-  const query = `country:us|postcode:${zip}`
-  const url = `https://maps.googleapis.com/maps/api/geocode/json?address=${encodeURIComponent(query)}&key=${apiKey}`
+  const query = `components=country:us|postal_code:${zip}`
+  const url = `https://maps.googleapis.com/maps/api/geocode/json?${query}&key=${apiKey}`
```

Comment:
It works on both ways but I'm writing this in this way to better match what's here: https://developers-dot-devsite-v2-prod.appspot.com/maps/documentation/geocoding/overview#component-filtering

Reply:
The country codes are always capitalized (e.g. "country:GB") in their example.